style.css

```diff
@@ -1 +1,366 @@
+.vegan {
+    object-fit: cover;
+    width: 100%;
+    max-height: 100%;
+}
+
+.hero {
+    grid-area: hero;
+}
+
+.feature-img {
+    object-fit: cover;
+    width: 100%;
+    max-height: 100%;
+}
+
+.product-container {
+    position: relative;
+    width: 100%;
+    height: auto;
+}
+
+.overlay {
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    height: 100%;
+    width: 100%;
+    opacity: 0;
+    transition: .5s ease;
+    background-color: #ffffff6e;
+}
+
+.product-container:hover .overlay {
+    opacity: 1;
+}
+
+.product-title {
+    color: #646262;
+    font-size: 20px;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    -webkit-transform: translate(-50%, -50%);
+    -ms-transform: translate(-50%, -50%);
+    transform: translate(-50%, -50%);
+    text-align: center;
+}
+
+.birthday {
+    grid-area: birthday;
+}
+
+.cupcakes {
+    grid-area: cupcakes;
+}
+
+.gluten-vegan {
+    grid-area: gluten;
+}
+
+.wedding {
+    grid-area: wedding;
+}
+
+
+/* FOOTER Section */
+
+.footer-container {
+    background-color: #cdafdb;
+    background-image: linear-gradient(to bottom right, #ecddf3, #ede7d2, #ffd5de, #e6dbeb, #ede7d2, #f9ddf6);
+    grid-area: footer;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    font-size: 1.1em;
+    font-weight: 500;
+}
+
+.footer-links {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+}
+
+.footer-links a {
+    font-size: 1.1em;
+    font-weight: 500;
+    color: #292b2f;
+    text-decoration: none;
+}
+
+a:hover {
+    color: #292b2f;
+    text-decoration: none;
+    cursor: pointer;
+}
+
+.footer-links a:hover {
+    cursor: pointer;
+}
+
+.newsletter-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+}
+
+.newsletter-container h5 {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    margin: 0;
+    padding-top: 30px;
+    font-size: 1.1em;
+    font-weight: 500;
+}
+
+.newsletter {
+    display: flex;
+    margin-top: 10px;
+}
+
+input[type="submit"] {
+    background-color: #617cbc;
+    border: 0;
+    border-radius: 5px;
+    cursor: pointer;
+    color: #fff;
+    font-size: 1rem;
+    font-weight: bold;
+    line-height: 1.4;
+    padding: 10px;
+    margin-left: 5px;
+}
+
+input[type="email"] {
+    border: 1px solid #ccc;
+    width: 200px;
+    padding-right: 40px;
+}
+
+#email::placeholder {
+    padding-left: 10px;
+}
+
+#email-label {
+    margin: 5px;
+}
+
+
+/* TABLET */
+
+@media screen and (min-width: 540px) {
+    body {
+        grid-template-columns: repeat(4, 1fr);
+        grid-template-rows: 120px 80px 340px 240px 200px;
+        grid-template-areas: "header     header     header     header" "nav        nav        nav        nav" "hero       hero       welcome    welcome" "birthday   cupcakes   wedding    gluten" "footer     footer     footer     footer";
```

Comment:
nice job using `grid-template-areas` for a different device size. If you arrange the grid areas in rows, it's easier to visualise the grid, e.g.
```
grid-template-areas: 
  "header     header     header     header" 
  "nav           nav           nav           nav" 
  "hero         hero         welcome   welcome" 
  "birthday  cupcakes wedding    gluten" 
  "footer      footer       footer        footer";
```